ssqatest/tests/frontend/latha_frontend_tests/test_50percent_coupon.py

```diff
@@ -12,10 +12,10 @@
 @pytest.mark.usefixtures("init_driver")
 
 class TestApply50PercentCoupon:
-    @pytest.mark.tcid40
-    @pytest.mark.pioneertcid40
-    @pytest.mark.lp001
-    @pytest.mark.esqf12
+@pytest.mark.tcid40
+@pytest.mark.pioneertcid40
+@pytest.mark.lp001
+@pytest.mark.esqf12

```

Comment:
O the indentation was fine. I was talking about removing empty line between the function and decorators. it should be indented like the function. 
just remove line 19 (the empty line. add empty line under the class definition.